### PR TITLE
[LIB-173] channel history implementation

### PR DESCRIPTION
**Pull request by jaskooo**

### CODE REVIEW

File: syncano-ios/SCChannel.m [hunk 1]
```diff
@@ -115,4 +116,26 @@ - (void)publishToChannelUsingAPIClient:(SCAPIClient *)apiClient withPayload:(NSD
...
 
+- (void)getChannelHistoryWithCompletion:(SCChannelHistoryCompletionBlock)completion {
+    [self getChannelHistoryUsingAPIClient:[Syncano sharedAPIClient] completion:completion];
+}
+- (void)getChannelHistoryFromSyncano:(Syncano *)syncano completion:(SCChannelHistoryCompletionBlock)completion {
+    [self getChannelHistoryUsingAPIClient:syncano.apiClient completion:completion];
+}
+- (void)getChannelHistoryUsingAPIClient:(SCAPIClient *)apiClient completion:(SCChannelHistoryCompletionBlock)completion {
+    NSString *path = [NSString stringWithFormat:@"channels/%@/history/",self.name];
+    [apiClient POSTWithPath:path params:nil completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
```
Comment (lifcio) [inline syncano-ios/SCChannel.m]: Channel history should use GET

File: syncano-ios/SCChannelHistoryResponse.m [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#import "NSObject+SCParseHelper.h"
+
+@implementation SCChannelHistoryResponse
+
+- (instancetype)initWithJSONObject:(id)JSONObject {
+    self = [super init];
+    if (self) {
+        self.prev = [JSONObject[@"prev"] sc_stringOrEmpty];
+        self.next = [JSONObject[@"prev"] sc_stringOrEmpty];
+        self.objects = [self parsedObjects:[JSONObject[@"objects"] sc_arrayOrNil]];
```
Comment (lifcio) [inline syncano-ios/SCChannelHistoryResponse.m]: Have you checked if backend always returns an array? Even if there is only 1 object? (it should)

**Comment by lifcio** [comment]
Other than above comments - looks good.

File: syncano-ios/SCChannelHistoryResponse.m (hunk 2 above)
Comment (jaskooo) [inline syncano-ios/SCChannelHistoryResponse.m]: Yes it is

**Pull request merged**